barriers/models/barriers.py

```diff
@@ -364,6 +364,27 @@ def latest_programme_fund_progress_update(self):
             return self.programme_fund_progress_updates[0]
         return None
 
+    @property
+    def start_date(self):
+        if self.data.get("start_date") is not None:
+            return dateutil.parser.parse(self.data.get("start_date"))
```

Comment:
Then here use the variable assigned in the conditional.

```suggestion
            return dateutil.parser.parse(start_date)
```